modules/configuring-firewall.adoc

```diff
@@ -80,10 +84,12 @@ If your environment has a dedicated load balancer in front of your {product-titl
 |===
 +
 --
-1. In a firewall environment, ensure that the `access.redhat.com` resource is on the allowlist. This resource hosts a signature store that a container client requires for verifying images when pulling them from `registry.access.redhat.com`.
+1. In a firewall environment, ensure that the `access.redhat.com` and `registry.access.redhat.com` resources are on the allowlist. These resources host a signature store that a container client requires for verifying images when pulling them from `registry.access.redhat.com`.
```

Comment:
Based on my other comment I think we can move this into the description of `access.redhat.com` and remove this node.